Documentation/ci-config/config-plugins-audit.sh

```diff
@@ -1,36 +1,62 @@
 #!/usr/bin/env bash
 
+##
+# Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 ##
 # @file config-plugins-audit.sh
 # @brief add plugin modules for a github repository
-# @see      https://github.sec.samsung.net/STAR/TAOS-CI
+# @see      https://github.com/nnsuite/TAOS-CI
 # @author   Geunsik Lim <geunsik.lim@samsung.com>
 
+##### Set environment for audit plugins
+declare -i idx=-1
 
 ###### plugins-base ###############################################################################################
-echo "[MODULE] plugins-base: Basis plugins with good quality"
+echo "[MODULE] plugins-base: Plugin group that follow Apache license with good quality"
```

Comment:
Is it ok that express apache?

Reply:
It's okay because TAOS-CI follows Apache license clause. However, let's remov the content in nnstreamer repository for avoid scanning procedure of the scan-code toolkit.

Reply:
Fixed